.github/CODEOWNERS

```diff
@@ -130,3 +133,45 @@ README*                                         @ceph/doc-writers
 /src/tools/rbd*                                 @ceph/rbd
 /systemd/rbdmap.service.in                      @ceph/rbd
 /udev/50-rbd.rules                              @ceph/rbd
+
+# rgw
+/doc/dev/radosgw                                @ceph/rgw @ceph/doc-writers
+/doc/radosgw                                    @ceph/rgw @ceph/doc-writers
+/examples                                       @ceph/rgw
```

Comment:
I'm a bit late with this comment but are you sure this works as intended, meaning that it doesn't take over the ownership of `examples/librados` or `examples/librbd` defined earlier in the file?

Perhaps rgw examples should be listed explicitly instead?

Reply:
that's what i assumed based on [this example](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners):
```
# In this example, @octocat owns any file in the `/apps`
# directory in the root of your repository except for the `/apps/github`
# subdirectory, as its owners are left empty.
/apps/ @octocat
/apps/github
```
but i tested a [change to /examples/librbd/Makefile](https://github.com/ceph/ceph/compare/main...cbodley:ceph:wip-github-librbd-examples?expand=1) and it tagged the rgw team instead

Reply:
i think we should just move all the rgw stuff under /examples/rgw, but i'm not sure whether any docs refer to the existing paths

Reply:
> but i tested a [change to /examples/librbd/Makefile](https://github.com/ceph/ceph/compare/main...cbodley:ceph:wip-github-librbd-examples?expand=1) and it tagged the rgw team instead

The order in which entries appear in the CODEOWNERS file matter:
```
# Order is important; the last matching pattern takes the most
# precedence.
```
If the rgw section came before the core and rbd sections, it would have worked.  It's exactly what I'm fixing in https://github.com/ceph/ceph/pull/50076 for docs.

> i think we should just move all the rgw stuff under /examples/rgw

Up to you!

Reply:
@idryomov thanks for spotting this

addressed in https://github.com/ceph/ceph/pull/50102